pkgs/os-specific/linux/bitcraze-udev-rules/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
```

No C compiler needed, and `rec` doesn't seem to be used

```suggestion
{ lib, stdenvNoCC, fetchurl }:

stdenvNoCC.mkDerivation {
```